app/models/compute_resources/foreman/model/vmware.rb

```diff
@@ -537,6 +564,18 @@ def associated_host(vm)
       associate_by("mac", vm.interfaces.map(&:mac))
     end
 
+    def display_type
+      attrs[:display] || 'vmrc'
```

Comment:
does it make sense to always default to vmrc? (all CR will default to that on first time?) do we have vmware version number so we can make an educated decision?

Reply:
Maybe we should add a migration to set all existing compute resources to `vnc` as that was the default behavior. As `vmrc` works with all compute resources, I think it's the preferred choice. We can get the vmware version number, but that requires an established connection and I think it's not worth the effort.